packages/desktop-client/src/components/modals/ImportTransactionsModal/Transaction.tsx

```diff
@@ -37,23 +60,34 @@ export function Transaction({
     [rawTransaction, fieldMappings],
   );
 
-  let amount, outflow, inflow;
-  if (rawTransaction.isMatchedTransaction) {
-    amount = rawTransaction.amount;
-    if (splitMode) {
-      outflow = amount < 0 ? -amount : 0;
-      inflow = amount > 0 ? amount : 0;
+  const { amount, outflow, inflow } = useMemo(() => {
+    if (rawTransaction.isMatchedTransaction) {
+      const amount = rawTransaction.amount;
+
+      return {
+        amount,
+        outflow: splitMode ? (amount < 0 ? -amount : 0) : null,
```

Comment:
Instead of null it used to be 0 - just checking this is intentional

Reply:
It was `null` before here too. See line 40.

Reply:
I've gone blind